src/components/lib/FlowGrid/FlowGrid.jsx

```diff
@@ -132,6 +165,10 @@ export default class FlowGrid extends Component{
     }
   }
 
+  componentWillUnmount() {
+    this.props.onDeSelectAll()
+  }
```

Comment:
This was a bit of future proofing, I guess/API separation. The space canvas also de-selects all upon unmount, but I think the FlowGrid should too. If you were to use this component, and it were to unmount, then the flowgrid has disappeared from the page; I don't think in that case it is sensible to retain which flowgrid locations were selected.